journals/apps/journals/models.py

```diff
@@ -401,6 +406,32 @@ def root_journal_page_url(self):
         descendants = self.get_descendants()
         return descendants[0].full_url if descendants else '#'
 
+    def get_journal_structure(self):
+        """ Returns hierarchy of the journal as a dict """
+        structure = {
+            "journal_structure": [
+                journal_page.specific.get_nested_children()
+                for journal_page
+                in self.get_children()
+            ]
+        }
+
+        return structure
+
+    def get_nested_children(self):
```

Comment:
I don't think you need to implement this method in JournalAboutPage as it's already implemented in JournalPage and will never get called on JournalAboutPage

Reply:
Oops did not realize it existed already! Yep I'll clean that up!